infrastructure/code/code_html.go

```diff
@@ -68,11 +108,91 @@ func getDetailsHtml(issue snyk.Issue) string {
 
 	dataFlowHtml := getDataFlowHtml(additionalData)
 
+	// Header
 	html := replaceVariableInHtml(detailsHtmlTemplate, "issueId", issue.ID)
 	html = replaceVariableInHtml(html, "issueTitle", additionalData.Title)
+	html = replaceVariableInHtml(html, "issueType", getIssueType(additionalData))
 	html = replaceVariableInHtml(html, "severityText", issue.Severity.String())
-	html = replaceVariableInHtml(html, "dataFlow", dataFlowHtml)
+	html = replaceVariableInHtml(html, "severityIcon", getSeverityIconSvg(issue))
+
+	html = replaceVariableInHtml(html, "issueOverview", additionalData.Message)
+	// Data flow
 	html = replaceVariableInHtml(html, "dataFlowCount", fmt.Sprintf("%d", len(additionalData.DataFlow)))
+	html = replaceVariableInHtml(html, "dataFlow", dataFlowHtml)
+
+	// External example fixes
+	html = replaceVariableInHtml(html, "repoCount", fmt.Sprintf("%d", additionalData.RepoDatasetSize))
+	html = replaceVariableInHtml(html, "exampleCount", fmt.Sprintf("%d", len(additionalData.ExampleCommitFixes)))
+	html = replaceVariableInHtml(html, "tabsNav", getTabsHtml(additionalData.ExampleCommitFixes))
 
 	return html
 }
+
+func getFileName(filePath string) string {
+	if filePath == "" {
+		return ""
+	}
+
+	// Windows paths
+	filePath = strings.Replace(filePath, "\\", "/", -1)
```

Comment:
I think, you can also use `ToSlash` for that.

Reply:
Adding this comment to the to IDE-231, thank you 🙏